src/src/jwt-verifier.ts

```diff
@@ -0,0 +1,43 @@
+import { Router } from 'express';
+import * as moment from 'moment';
+const router = Router();
+export default router;
+
+const jws = require('jws');
+const fs = require('fs');
+
+const verify = function (req, res, next) {
+    let detachedSignature = req.headers['x-jws-signature'];
+    console.log("detachedSignature: " + detachedSignature);
+    if (!detachedSignature) {
+        console.error("detachedSignature is falsy");
+        res.status(401).send({
+            "responseHeader": {
+                "requestId": req.headers.requestId,
+                "sendDate": moment().toISOString(),
+                "isCallback": true
+            },
+            "code": "INCORRECT_DETACHED_SIGNATURE",
```

Comment:
I think we do not need to use word `deatached` in error messages, it is simply the signature.